### DISCUSSION TITLE
Unfairness in level distribution.

### DISCUSSION THREAD

**Discussion by amishhhaaaa** [Akshat111111/Hedging-of-Financial-Derivatives#251] [General]
@Akshat111111 I have noticed tags are assigned to the PRs now and assigning level 2 for model deployment and level 2 for visualisations is unfair as it takes a lot of research, trial errors for which model gives a better result to documentation and plots which we did in model deployment but creating visualisation is a rather simple task where without even thoroughly going through the entire work it hardly takes 10-15 minutes to create them(visualisations that aren't even explained). I would appreciate if you acknowledge this issue.
Also some PRs of visualisation were wrong and the contributor was still given points for it when it should rather have been  closed.

**Comment by Akshat111111** [comment]
> @Akshat111111 I have noticed tags are assigned to the PRs now and assigning level 2 for model deployment and level 2 for visualisations is unfair as it takes a lot of research, trial errors for which model gives a better result to documentation and plots which we did in model deployment but creating visualisation is a rather simple task where without even thoroughly going through the entire work it hardly takes 10-15 minutes to create them(visualisations that aren't even explained). I would appreciate if you acknowledge this issue. Also some PRs of visualisation were wrong and the contributor was still given points for it when it should rather have been closed.

you are free to give your suggestions on other people work and I also appreciate it.The level is decided on various things like problem statement, idea and work as well as documentation.The level also depends on consistency and engagement.

**Reply by amishhhaaaa** [reply]
Thank you so much for clarifying however I am still not able to get how creating 2-3 plots considered as hard as the number of steps included in model deployment to be given the same level 1)data preprocessing (cleaning and aligning Xcel data) searching and incorporating relevant data from various resources 2)model deployment and analysis of which one gives better results feature comparisons 3) writing documentation definitions etc.4) creating plots which support the work. 
Model deployment requires 50-100 lines of code whereas plots only require 5-10 lines of code without any deep understanding. 
Looking forward to your guidance.

**Comment by Amarta113** [comment]
@Akshat111111 Is it fair to assign multiple issues to one contributor before they have completed the first one?

**Reply by Akshat111111** [reply]
i have done it inorder to safeguard their idea,but after a certain period, if they fail, then I will unassign
